src/encoded/schemas/static_section.json

```diff
@@ -37,6 +37,7 @@
             "title": "Section Type",
             "type": "string",
             "description": "What this section is used for. Defaults to 'Page Section'.",
+            "permission" : "import_items",
```

Comment:
Should prevent editing of that field unless admin (have permission to mass-import items).